tests/python/gaia-ui-tests/gaiatest/tests/functional/browser/test_browser_bookmark.py

```diff
@@ -17,6 +17,8 @@ def setUp(self):
         GaiaTestCase.setUp(self)
         self.connect_to_local_area_network()
         self.apps.set_permission_by_url(Search.manifest_url, 'geolocation', 'deny')
+        # not sure why the Smart Collection geolocation is being prompted.
+        self.apps.set_permission_by_url(Collection.manifest_url, 'geolocation', 'deny')
```

Comment:
The search app for 2.1 and below uses geolocation. 2.2 they are removing the requirement, I think.  Do we have branch versions for tests as well?